src/tools/kg_tools.py

```
@@ -142,14 +144,144 @@ async def crawl_repo(ctx: Context, repo_url: str) -> str:
+                                }
+                            )
+                            section_node.add_relationship("PART_OF", f"{repo_url}#{chunk.source_path}")
+                            enhanced_content.structural_nodes.append(section_node)
+                
+                # Store enhanced structure in Neo4j
+                if enhanced_content.structural_nodes:
+                    await neo4j_ingester.ingest(enhanced_content)
+                    enhanced_nodes_created = len(enhanced_content.structural_nodes)
+                    logger.info(f"✓ Stored {enhanced_nodes_created} enhanced documentation nodes in Neo4j")
+            
+        except Exception as qdrant_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)